### DISCUSSION TITLE
Using Wordplate with Local by  flywheel ?

### DISCUSSION THREAD

**Discussion by steveprn** [wordplate/wordplate#337] [Q&A] [answered]
Hello @vinkla !

I use Local ( flywheel ) to make my wordpress locally, but I was wondering if it was possible to use it with Wordplate?

Could someone please fill me in on this?

Thanks in advance !

**Comment by vinkla** [comment, marked as answer]
Hey Beda!

I haven't used Local myself, but I found this:

https://community.localwp.com/t/setup-bedrock-site/16513/16
https://discourse.roots.io/t/local-bedrock-development-with-local-by-flywheel/14535

Note that you'll have to replace `/wp` with `/wordpress` since WordPlate uses the latter.

**Reply by steveprn** [reply]
Thanks for your answer @vinkla !

It seems tedious to me to put all this in place, I will try to work without Local

**Reply by vinkla** [reply]
I can personally recommend Laravel Valet. There is a driver available in the documentation.

**Reply by steveprn** [reply]
Hi again @vinkla,
 I manage to install wordplate with composer but everytime I  try to create the project, the terminal says he can't install the different package ? Did I do something wrong ?
